lib/ansible/module_utils/powershell.ps1

```diff
@@ -105,40 +105,81 @@ Function Fail-Json($obj, $message = $null)
 #Get-AnsibleParam also supports Parameter validation to save you from coding that manually:
 #Example: Get-AnsibleParam -obj $params -name "State" -default "Present" -ValidateSet "Present","Absent" -resultobj $resultobj -failifempty $true
 #Note that if you use the failifempty option, you do need to specify resultobject as well.
-Function Get-AnsibleParam($obj, $name, $default = $null, $resultobj, $failifempty=$false, $emptyattributefailmessage, $ValidateSet, $ValidateSetErrorMessage)
+Function Get-AnsibleParam($obj, $name, $default = $null, $resultobj, $failifempty=$false, $emptyattributefailmessage, $ValidateSet, $ValidateSetErrorMessage, [string[]]$aliases, [Parameter(DontShow)]$OriginalName)
```

Comment:
DontShow bombs in PowerShell 3 with: 
Property 'DontShow' cannot be found for type 'System.Management.Automation.ParameterAttribute'.
At line:2 char:5
+     [Parameter(DontShow)] $broken
+     ~~~~~~~~~~~~~~~~~~~~~
    + CategoryInfo          : InvalidOperation: (:) [], ParentContainsErrorRecordException
    + FullyQualifiedErrorId : PropertyNotFoundForType